### ISSUE TITLE
[Question/Issue] Prometheus stop working when the computer is in sleeping mode

### ISSUE THREAD

**Issue by JeromeVi** [prometheus/prometheus#2432]
**What did you do?**
I run Prometheus in Docker, 1Prometheus on my local and 1 in a Docker machine called `m1`.
I putted my computer in sleeping mode and this morning I went to my Grafana website (where I pick data from the 2 Prometheus DB) and the only one who was working was the one of `m1`.

**What did you expect to see?**
The data from the 2 DB 

**What did you see instead? Under which circumstances?**
Only the data of Prometheus from `m1`

**Environment**
* System information:

	macOS Sierra running Docker, in Docker it's Darwin 16.4.0 x86_64

* Prometheus version: 

	1.5.0

* Alertmanager version:

   0.5.1
* Prometheus configuration file:
```
global:
  scrape_interval:     15s
  evaluation_interval: 15s

  # Attach these labels to any time series or alerts when communicating with
  # external systems (federation, remote storage, Alertmanager).
  external_labels:
      monitor: 'docker-host-alpha'

# Load and evaluate rules in this file every 'evaluation_interval' seconds.
rule_files:
  - "targets.rules"
  - "host.rules"
  - "containers.rules"

# A scrape configuration containing exactly one endpoint to scrape.
scrape_configs:
  - job_name: 'nodeexporter'
    scrape_interval: 5s
    static_configs:
      - targets: ['nodeexporter:9100']

  - job_name: 'cadvisor'
    scrape_interval: 5s
    static_configs:
      - targets: ['cadvisor:8080']

  - job_name: 'prometheus'
    scrape_interval: 10s
    static_configs:
      - targets: ['localhost:9090']
```

* Logs:
The logs from the local Prometheus were from yesterday when I putted my computer in sleeping mode and the ones of `m1` were from today so that's right !

# How to solve 

To solve that I removed all the containers and created them again

**Comment by danni-m** [comment]
One of the issues i've experienced with docker-for-mac was a time drift caused by a bug in docker (https://github.com/docker/for-mac/issues/17).
Restarting the docker-machine fixed it (I was using version 1.4 - 1.5 versions).

**Comment by JeromeVi** [comment]
@danni-m yes if I restart the machine it works that's right. So I'll check this link your gave me and thank you :)
